src/vdom.rs

```diff
@@ -195,29 +197,67 @@ impl<Ms, Mdl, ElC: View<Ms> + 'static, GMs: 'static> App<Ms, Mdl, ElC, GMs> {
         }
     }
 
+    /// Bootstrap the dom with the vdom by taking over all children of the mount point and
+    /// replacing them with the vdom if requested. Will otherwise ignore the original children of
+    /// the mount point.
     fn bootstrap_vdom(&self) -> El<Ms> {
+        let mount_type = self.cfg.mount_type.borrow().unwrap_or(MountType::Append);
         // "new" name is for consistency with `update` function.
         // this section parent is a placeholder, so we can iterate over children
         // in a way consistent with patching code.
         let mut new = El::empty(dom_types::Tag::Placeholder);
 
+        // Map the DOM's elements onto the virtual DOM if requested to takeover.
+        if mount_type == MountType::Takeover {
+            // Construct a vdom from the root element. Subsequently strip the workspace so that we
+            // can recreate it later - this is a kind of simple way to avoid missing nodes (but
+            // not entirely correct).
+            // TODO: 1) Optimize by utilizing a patching strategy instead of recreating the workspace
+            // TODO: nodes.
+            // TODO: 2) Watch out for elements that should not be recreated, such as script nodes,
+            // TODO: and other, similar things. For now, leave the warning in the builder's
+            // TODO: documentation.
```

Comment:
Could you move this comment block into a new GitHub issue and just add a shorter comment with todo and link to that issue here?
So we don't forget to fix it and you can describe the problem / solution in more details (if you want) while you have details still in your head :)

Reply:
Got it.

Reply:
#277